src/app/app-loading.pug

```diff
@@ -1,4 +1,3 @@
-#spash(style='height: 100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')
-  div(style='width: 80%; max-width: 150px')
+#spash(style='z-index:100; background:#fff; position:fixed; bottom: 0; right: 0; left: 0; top: 0; height:100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')  
```

Comment:
where does this file `app-loading.pug` load? I see that you removed `include ../src/app/app-loading.pug`

Reply:
@piavgh It is for `#wrap(ng-include='session.template')`. If this file is removed, the default loading animation from common libs will be used instead